### PR TITLE
Add home page with description

### LINKED ISSUE #1: Implement home page
**Short description**
We need a landing page that is the first thing the user sees when opening the index page.

**Acceptance Criteria**
Should route here when going to domain page
Should have a header where it says Oles eleven
Should have a description of what oles eleven is.

### PR DESCRIPTION
**Pull request by OleVedeler**
Fixes #1

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -14,46 +12,18 @@ export default function Web() {
...
-              <div key={singleItem.title} className="flex flex-col items-center justify-center text-center">
-                <div className="mb-4 flex h-10 w-10 items-center justify-center rounded-full bg-primary-100 p-1.5 text-blue-700 dark:bg-primary-900 lg:h-12 lg:w-12">
-                  {singleItem.icon}
-                </div>
-                <h3 className="mb-2 text-xl font-bold dark:text-white">{singleItem.title}</h3>
-                <p className="text-gray-500 dark:text-gray-400">{singleItem.description}</p>
-              </div>
-            ))}
+            <h1 className="mb-4 max-w-2xl text-4xl font-extrabold leading-none tracking-tight dark:text-white md:text-5xl xl:text-6xl">Oles eleven</h1>
+            <p className="mb-6 max-w-2xl font-light text-gray-500 dark:text-gray-400 md:text-lg lg:mb-8 lg:text-xl">This website is a a challenge page where there will be posted 11 challenges for Ole and Benjamin to complete. </p>
```
Comment (Benjamsk) [inline pages/index.tsx]: a a should be a

**Review by Benjamsk — CHANGES_REQUESTED** [review]

**Comment by Benjamsk** [comment]
Please 
run `yarn prettier`
see output
run `yarn prettier:fix`
see output
run `yarn prettier`

**Comment by Benjamsk** [comment]
Test failing, should not fail =)

File: pages/index.tsx (hunk 1 above)
Comment (OleVedeler) [reply pages/index.tsx]: fixed

**Review by Benjamsk — APPROVED** [review]

**Pull request merged**